cognite/neat/core/_data_model/models/physical/_validation.py

```
@@ -391,7 +391,9 @@ def _consistent_container_properties(self) -> IssueList:
                         "rows",
                     )
                 )
-            index_definitions = {",".join(prop.index) for _, prop in properties if prop.index is not None}
+            index_definitions = {
+                ",".join([index.suffix for index in prop.index]) for _, prop in properties if prop.index is not None
```

if type of index is not explicitly set, we need to raise warning or we need to clearly state in docs that if one is not setting this, we will automatically set it to the default index type.